src/main/resources/application.properties

```diff
@@ -479,4 +479,5 @@ sample.bulk.upload.designation.file.name=bulkUploadDesignationMasterData.xlsx
 org.designation.bulk.upload.container.name=orgDesignation
 kafka.topics.org.designation.bulk.upload.event=dev.org.designation.bulk.upload
 kafka.topics.org.designation.bulk.upload.event.group=orgDesignationBulkUpload
-redis.master.data.read.timeout=3600
\ No newline at end of file
+redis.master.data.read.timeout=3600
+app.default.departmentId=01390354700029132834
```

Comment:
this new param is not required and do not hardcode the value iGOT too... ... use the following... 

user.registration.custodian.orgId=0131397178949058560
user.registration.custodian.orgName=igot